src/Scheduler/SchedulerConfig.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Scheduler;
+
+use InvalidArgumentException;
+use ReflectionMethod;
+use Tempest\Console\ConsoleCommand;
+
+final class SchedulerConfig
+{
+    public function __construct(
+        public string $path = "php tempest",
+        public string $output = "/dev/null",
```

Comment:
Do `$output` and `$outputMode` have to be configurable?

Reply:
I think that they do have to be configurable - if the command's execution fails, we'd want to see what went wrong and be able to debug whether or not they executed at all, it's a configuration setting in every other project as far as I'm aware, I do believe that it should live at the #[Schedule] level though, where it was originally, but after your mixed feelings about it, this felt like a nice compromise :D 

Lemme know how you want to move forward with it

Reply:
Well now that you explained it like this, I agree that it makes most sense to have it on the `Schedule` attribute. So sorry for changing my mind 😅 😁